### PR TITLE
[ManyToonKR] add website support

### LINKED ISSUE #1527: Site Request: manytoon.club (Korean Raw)
https://manytoon.club/

### PR DESCRIPTION
**Pull request by dodieboy**
Resolves #1527

### CODE REVIEW

File: src/web/mjs/connectors/ManyoonKR.mjs [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import WordPressMadara from './templates/WordPressMadara.mjs';
+
+export default class ManyToon extends WordPressMadara {
```
Comment (XenonR) [inline src/web/mjs/connectors/ManyoonKR.mjs]: The classname `ManyToon` is alredy used.

**Review by XenonR — CHANGES_REQUESTED** [review]
Classname is duplicate.

File: src/web/mjs/connectors/ManyoonKR.mjs (hunk 1 above)
Comment (27136773) [reply src/web/mjs/connectors/ManyoonKR.mjs]: There seems to be a typo in the filename too

**Review by XenonR — APPROVED** [review]
All good.

**Pull request merged**